Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -82,31 +82,24 @@ - (NSUInteger)hash
 
 #pragma mark -
 
+NSString *const kNSNotificationName_LocalProfileUniqueId = @"kNSNotificationName_LocalProfileUniqueId";
```

Comment:
No, I'm not worried about the prefix convention, which I think is good.

But doesn't look like we're using this as a notification name. Isn't this constant used only as the uniqueId of a DB object?